src/pages/smart-collection/store/sagas.ts

```diff
@@ -29,8 +30,12 @@ const COLLECTIONS_LIMIT = 25
 function* fetchHeavyRotation() {
   const topListens = yield call(Explore.getTopUserListens)
 
+  const users = yield call(
+    retrieveUsers,
+    topListens.map((t: any) => t.userId)
+  )
   const trackIds = topListens
-    .filter((track: UserTrack) => !track.user.is_deactivated)
+    .filter((track: any) => !users[track.userId].is_deactivated)
```

Comment:
I think type should be `TopUserListen` from `/services/audius-backend/Explore.ts`

Reply:
Hmmm, still seeing the error, maybe check if track user id is in users maps
ie. `.filter((track: any) => !(users[track.userId]?.is_deactivated ?? true))`